src/translations/sqlb_zh.ts

```diff
@@ -4864,32 +4865,32 @@ If you answer no we will attempt to import the data in the SQL file to the curre
     <message>
         <location filename="../MainWindow.cpp" line="327"/>
         <source>Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>窗口布局</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="336"/>
         <source>Simplify Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>精简窗口布局</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="337"/>
         <source>Shift+Alt+0</source>
-        <translation type="unfinished"></translation>
+        <translation>Shift+Alt+0</translation>
```

Comment:
We are asking to contributors not to translate shortcuts on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts).

There was a discussion about this in PR #2406 some time ago, but in that case,
the shortcut didn't work and we wasn't sure if that was a Qt bug, so we proceeded the merge without deletion.

If possible, delete the shortcut translation, and if the shortcut doesn't work after deleting,
please tell us so we can just merge it without deletion.

Reply:
There are over 100 translated strings of shortcut in this translation file. Should and how will them all be removed?

Reply:
Hmm.. Never mind. After approve this changes and merge this PR, and I will delete them. :)